docs/presentation/advanced/Caching.md

```diff
@@ -0,0 +1,65 @@
+# Caching
+
+The presentation backend uses a number of caches to improve performance. This page provides a list of caches that may have a notable effect on either disk or memory.
+
+## Primary connection schema cache
+
+Each iModel contains a number of ECSchemas which consist of ECClass, ECProperty and other entity definitions (see [ECSchema page](../../bis/ec/ec-schema.md) for more details). All of that is used to describe the data stored in an iModel and is leveraged by the presentation library to create efficient ECSQL queries based on given presentation rules.
+
+There's an in-memory schema cache associated with each iModel to improve performance of getting that meta-data. Any time a new meta-data entity is requested to be loaded, it's also put into the cache. The cache is unbounded in size and may grow as much as necessary to cache all schemas, possibly reaching several hundred megabytes for iModels with many large schemas. It's only released when the associated iModel is closed.
+
+## Worker connections cache
+
+Presentation manager uses worker threads to fulfil requests to avoid blocking the main thread (see [PresentationManagerProps.workerThreadsCount]($presentation-backend)). Each worker thread uses a separate SQLite connection to the iModel to be able to run multiple queries in parallel. Each SQLite connection uses an in-memory page cache to increase query performance (see [SQLite's cache_size documentation](https://www.sqlite.org/pragma.html#pragma_cache_size)), which by default is configured at 32 MB. This means that the total amount for memory these caches may consume is:
+
+`{number of iModels used for requests} * {number of worker threads} * {worker connection cache size}`
+
+The cache is released when the worker connection is closed, which happens when either the iModel is closed or the presentation manager is disposed.
+
+Size of the cache can be controlled when initializing [Presentation]($presentation-backend) or creating [PresentationManager]($presentation-backend) by passing a custom value for [PresentationManagerProps.caching.workerConnectionCacheSize]($presentation-backend).
```

Comment:
Mention these enhancements in NextVersion.md and link to this .md document?

Reply:
Added a section to NextVersion.md

Reply:
Thanks. Nice docs.